src/async-components/views/dialogs/security/ExportE2eKeysDialog.tsx

```diff
@@ -63,21 +68,40 @@ export default class ExportE2eKeysDialog extends React.Component<IProps, IState>
         this.unmounted = true;
     }
 
-    private onPassphraseFormSubmit = (ev: React.FormEvent): boolean => {
-        ev.preventDefault();
+    private async verifyFieldsBeforeSubmit(): Promise<boolean> {
+        const fieldIdsInDisplayOrder = [this.fieldPassword, this.fieldPasswordConfirm];
```

Comment:
these are actual fields, not IDs?

```suggestion
        const fieldsInDisplayOrder = [this.fieldPassword, this.fieldPasswordConfirm];
```